samples/bluetooth/mesh/chat/README.rst

```diff
@@ -140,7 +140,7 @@ Interacting with the sample
 After completing the steps above, a command can be sent to the sample.
 The sample supports the following commands:
 
-chat help
+chat --help
```

Comment:
I guess it's true what they say, it's easy to get blind after reading through something for a while :)
I see embrance being used instead of embrace in line 155 and 159. Would you mind fixing that as well?

Reply:
This is the time when it's good to be a rookie. I actually tried to type the command as written in the doc :) Np, I'll fix that.

Reply:
Isn't wrap better? Embrace doesn't sound like the right word for this IMO.

Reply:
I agree.

Reply:
Hm... I am not sure that I like using wrap... It is a developer expression used in coding, more than a "proper English", so I am leaning towards enclose here.
If embrace is to be replaced.
Your final call, David :)